deployment/v3/testrig/apitestrig/install.sh

```
@@ -30,6 +30,15 @@ function installing_apitestrig() {
   API_INTERNAL_HOST=$( kubectl -n default get cm global -o json  |jq -r '.data."mosip-api-internal-host"' )
   ENV_USER=$( kubectl -n default get cm global -o json |jq -r '.data."mosip-api-internal-host"' | awk -F '.' '/api-internal/{print $1"."$2}')
 
+  read -p "Please enter the DB port (press Enter to use default 5432, or enter custom port for external PostgreSQL) : " db_port
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)